phpunit.xml

```diff
@@ -1,30 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit
-        colours = "true"
-        convertErrorsToExceptions   = "true"
-        convertNoticesToExceptions  = "true"
-        convertWarningsToExceptions = "true"
-        strict = "true"
-    >
-
-    <filter>
-        <whitelist>
-            <directory suffix=".php">symfony</directory>
-            <exclude>
-                <directory>symfony/apps</directory>
-                <directory>symfony/cache</directory>
-                <directory>symfony/config</directory>
-                <directory>symfony/data</directory>
-                <directory>symfony/lib</directory>
-                <directory>symfony/web</directory>
-            </exclude>
-        </whitelist>
-    </filter>
-
-    <testsuites>
-        <testsuite name="All Tests">
-            <file>symfony/test/PluginAllTests.php</file>
-        </testsuite>
-    </testsuites>
-
-</phpunit>
\ No newline at end of file
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="symfony/test/util/bootstrap.php" colors="true" convertErrorsToExceptions="true" convertNoticesToExceptions="true" convertWarningsToExceptions="true" beStrictAboutOutputDuringTests="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
+  <coverage>
+    <include>
+      <directory suffix=".php">symfony</directory>
+    </include>
+    <exclude>
+      <directory>symfony/apps</directory>
+      <directory>symfony/cache</directory>
+      <directory>symfony/config</directory>
+      <directory>symfony/data</directory>
+      <directory>symfony/lib</directory>
+      <directory>symfony/web</directory>
+    </exclude>
+  </coverage>
+  <testsuites>
+    <testsuite name="All Tests">
+      <!--            <file>symfony/test/PluginAllTests.php</file>-->
+      <file>symfony/plugins/orangehrmAdminPlugin/test/model/service/JobTitleServiceTest.php</file>
```

Comment:
That may work with 
```
*/test/*Test.php
```

Reply:
`symfony/test/PluginAllTests.php` also working. But the initial point it's breaking all unit tests. Once finish with the admin module we can replace these files with `symfony/plugins/orangehrmAdminPlugin/test/orangehrmAdminPluginAllTests.php` once done with all modules we can replace `symfony/test/PluginAllTests.php`.